src/components/Auth/ActivationForm/index.tsx

```diff
@@ -40,10 +39,21 @@ function ActivationForm() {
     <Container>
       <FormContainer>
         <Form>
-          <AuthConfirmationContainer>
-            <AuthConfirmationImg src={checkmark} />
-          </AuthConfirmationContainer>
-          <Text>{t('Auth.activationText')}</Text>
+          {isActiveLink ? (
+            <>
+              <AuthConfirmationContainer>
+                <AuthConfirmationImg src={checkmark} />
+              </AuthConfirmationContainer>
+              <Text>{t('Auth.activationText')}</Text>
+            </>
+          ) : (
+            <>
+              <AuthConfirmationContainer>
+                <AuthConfirmationImg src={emotionSad} />
+              </AuthConfirmationContainer>
+              <Text>{t('Auth.activationTextError')}</Text>
+            </>
```

Comment:
why dont you make reusable component, you have same block twice

Reply:
they have different pictures and texts but anyway made some changes to make code more attractive